dotCMS/src/integration-test/java/com/dotcms/publishing/PublisherAPITest.java

```diff
@@ -319,4 +326,293 @@ private FileFilter getNoBundleXMLFileFilter() {
         return file -> !file.getAbsolutePath().endsWith("bundle.xml");
     }
 
+    @Test
+    public void test_addFilter(){
+        publisherAPI.getFilterDescriptorMap().clear();
+
+        final Map<String,Object> filtersMap =
+                ImmutableMap.of("dependencies",true,"relationships",true,"excludeClasses","Host,Workflow");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "relationships" appears 6 times in this file; the first occurrence is on line 330](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)